src/Jira/Api.php

```diff
@@ -173,14 +173,14 @@ protected function clearLocalCaches()
 	/**
 	 * Get fields definitions.
 	 *
-	 * @return array
+	 * @return array|false
```

Comment:
On second thought its a bit dodgy to make a BC change without adding tests as well. I might add some tomorrow